DataFormats/METReco/src/MET.cc

```diff
@@ -68,6 +68,24 @@ MET::MET( double sumet_, const std::vector<CorrMETData>& corr_,
     }
 }
 
+// Copy constructor
+//____________________________________________________________________________||
+MET::MET( const MET& met ): 
```

Comment:
If the default constructor works, better remove this overload.
This will reduce code maintenance needs if new variables are added/removed.